### PR TITLE
Add SAP-HANA known issues

### PR DESCRIPTION
**Pull request by nimatrueway**
Add known issues for SAP-HANA database support

### CODE REVIEW

**Comment by nimatrueway** [comment]
[here](https://gitlab.rnd.mendix.com/appdev/appdev/blob/dat/feature/sap-hana-integration/runtime/datastorage-connectionbus/src/com/mendix/connectionbus/connections/jdbc/implementations/saphana/ImplementationStatus.md) is the actual technical documentation going through rigorous details and tricks of SAP-HANA.

@grootjans @ishan1729 Please let me know if I missed anything, or the sentences need correction.

**Comment by MarkvanMents** [comment]
Hi @nimatrueway,
Please note in future that you should fork the **development** branch and not *master* as we always do a build there to check. This is documented in https://docs.mendix.com/developerportal/community-tools/contribute-to-the-mendix-documentation but I'll look into making it even clearer.
I will make the change to the base branch for this PR.
Mark

**Comment by nimatrueway** [comment]
Hey @MarkvanMents,
My apologies, and thank you.

File: content/refguide/saphana.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: "SAP HANA"
+parent: "data-storage"
+---
+
+## Known Issues
+
+### Mandatory Limiting rows after specify an OFFSET
```
Comment (ishan1729) [inline content/refguide/saphana.md]: Is this header also going to go? If not then we should use `specifying` iso `specify`.

Comment (nimatrueway) [reply content/refguide/saphana.md]: Yes @ishan1729, these are the items that are "to-be-removed" since I thought does not concern Mendix Users.

Comment (ishan1729) [reply content/refguide/saphana.md]: Great!

**Comment by ishan1729** [comment]
@nimatrueway Can't seem to know as to where this document will be finally published. But, IMO this should be placed under the main Datastorage page(https://docs.mendix.com/refguide/data-storage) like we have for Mysql etc.. 

Ignore my comment if that is already the case.

**Comment by nimatrueway** [comment]
Dear @MarkvanMents !
I read the contributors' document and could not figure out how to add an item in doc's navigation bar. Can you help me a bit, give me a hint maybe?

**Comment by MarkvanMents** [comment]
@nimatrueway - by setting the parent to `data-storage` the title of the document will be inserted into the navigation automatically as a child of data-storage. There is nothing else you need to do.

We can also control the order of the documents in the section if you want it to appear in a particular place. But this is usually the last thing we do after merging and seeing what it looks like on the ACCP site.

**Comment by grootjans** [comment]
@MarkvanMents what still needs to happen to finalize this PR?

**Comment by MarkvanMents** [comment]
@grootjans If you are happy with it, I'll proofread it and publish it.
Can you confirm that the technical details are correct?
Thanks

**Pull request merged**